installscripts/jazz-terraform-unix-noinstances/chefProvisioner.tf

```diff
@@ -79,12 +79,12 @@ resource "null_resource" "chef_provision_jenkins_server" {
   provisioner "remote-exec" {
     inline = [
       "sudo sh ~/cookbooks/installChef.sh",
-      "curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq",
+      "sudo curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 -o /usr/local/bin/jq",
       "sudo chmod 755 /usr/local/bin/jq",
       "cat ~/cookbooks/jenkins/files/plugins/plugins0* > plugins.tar",
       "chmod 777 plugins.tar",
       "sudo tar -xf plugins.tar -C /var/lib/jenkins/",
-      "curl -O https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py",
+      "sudo curl -O https://bootstrap.pypa.io/get-pip.py && sudo python get-pip.py",
```

Comment:
You should not need the `sudo` command here on the left-hand side `curl` command, as you are fetching it to the local directory, which the current user should have write access to, unless we are doing something strange that we should not be doing (like running the `chef` cookbooks in the root user's home folder as a non-root user)

Can you verify that the problem is not simply that there is an old, existing `get-pip.py` file that needs to be cleaned up, that was previously fetched as root, and so is not overwritable by non-root?

Reply:
yes. there was an existing file which is under the ownership - root. Reverted the change.